tools/commentMonitor/main.go

```diff
@@ -141,9 +149,24 @@ func regexValidation(regex string, comment string) ([]string, error) {
 }
 
 func writeArgs(args []string, output string) {
+	// More regex can be added.
+	regex := make(map[string]*regexp.Regexp)
+	// Regex for PR number.
+	regex["PR_NUMBER"] = regexp.MustCompile("(?m)^([0-9]+)\\s*$")
+	// Regex for Release version.
+	regex["RELEASE"] = regexp.MustCompile("(?mi)^(master|[0-9]+\\.[0-9]+\\.[0-9]+\\S*)\\s*$")
```

Comment:
the `v` prefix was made mandatory when commenting to avoid confusion in the future.
https://github.com/prometheus/prombench#trigger-tests-via-a-github-comment

I think this regex will need the little bit of modification for that, also do we need double backslash when using it inside a gofile? `\\` I think this should work, not 100% sure though:
```
(?mi)^(master|v[0-9]+\.[0-9]+\.[0-9]+\S*)\s*$
```

Reply:
Build fails if `\` used instead of `\\`

Reply:
Worked after using \` instead of `"`